src/test/java/com/faforever/client/chat/ChatChannelUserItemControllerTest.java

```diff
@@ -200,11 +206,12 @@ public void testNullGameSetsStatusToNothing() {
     player.setGame(GameBuilder.create().defaultValues().host("junit").state(GameStatus.PLAYING).get());
     WaitForAsyncUtils.waitForFxEvents();
 
-    assertThat(instance.statusLabel.getText(), is("Playing"));
+    assertTrue(instance.playerStatusIndicator.isVisible());
+    assertThat(instance.playerStatusIndicator.getImage().getUrl(), endsWith("playing.png"));
     player.setGame(null);
     WaitForAsyncUtils.waitForFxEvents();
 
-    assertThat(instance.statusLabel.getText(), is(""));
+    assertFalse(instance.playerStatusIndicator.isVisible());
```

Comment:
assertThat(...,is(false))

Reply:
why the heck should I do that? There's a method for that purpose. Even intellij automatically detects that and advises me to convert to assertFalse.

Reply:
well ok then